dune-project

```diff
@@ -7,18 +7,17 @@
 (authors "Jane Street Group, LLC <opensource@janestreet.com>")
 (source (github ocaml/dune))
 
-(opam
- (package
-  (name dune)
-  (depends
-   (ocaml (>= 4.02))
-   base-unix
-   base-threads)
-  (conflicts
-   (jbuilder (<> "transition"))
-   (odoc (< 1.3.0)))
-  (synopsis "Fast, portable and opinionated build system")
-  (description "
+(package
+ (name dune)
```

Comment:
Should we comment in the docs on the role of the global `(name)` field if there exists a `(package (name))` field?

Reply:
I think the global `name` field is already documented